### DISCUSSION TITLE
How to use Firebase on Flutter and Swift?

### DISCUSSION THREAD

**Discussion by hahai96** [firebase/flutterfire#7651] [Q&A]
I'm using FireStore on Flutter and Swift, but when I build I get an error `Terminating app due to uncaught exception 'FIRIllegalStateException', reason: 'Firestore instance has already been started and its settings can no longer be changed. You can only set settings before calling any other methods on a Firestore instance.'
`
This seems to be because Flutter initialized the Firestore with the statement **Firebase.initializeApp**.  Has anyone encountered this problem, please help

**Comment by akshatshah-1** [comment]
@hahai96 did you find any solution for this? I have the same issue.
